.golangci.bck.yaml

```diff
@@ -0,0 +1,99 @@
+issues:
```

Comment:
I was a little confused about this backup file until I read up on `golangci-lint migrate` and realized this was probably auto-generated to preserve a copy of the old config. Could we leave a comment about it being autogenerated or get rid of it altogether if not necessary? Leaning toward getting rid of it altogether myself.

Reply:
Good call. Ive removed it